.github/CODEOWNERS

```
@@ -13,7 +13,7 @@ Cargo.toml @MichaelMauderer @4e6 @mwu-tow @farmaazon
 # GUI
 /run @MichaelMauderer @wdanilo
 /build/ @MichaelMauderer @wdanilo
-/app/gui/ @MichaelMauderer @wdanilo @farmaazon @mwu-tow
+/app/gui/ @MichaelMauderer @wdanilo @farmaazon @mwu-tow @jdunkerley
```

As I'm a bit tired of being spammed with PRs which does not affect my area of knowledge, I'd consider removing the ownership from the CHANGELOG; usually someone competent should review PRs anyway.
```suggestion
/app/gui/ @MichaelMauderer @wdanilo @farmaazon @mwu-tow
/app/gui/CHANGELOG.md
```